app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js

```diff
@@ -0,0 +1,229 @@
+// @flow
```

Comment:
Is it a good idea to hide these screen components somewhere in the package? I created `navigation` folder for this (but not sure what is better).

Reply:
I think I will move these modal screens also to the navigation folder. Makes sense.